mod/logbrowser/views/default/logbrowser/refine.php

```diff
@@ -5,36 +5,26 @@
  * @package ElggLogBrowser
  */
 
-$form_vars = array(
+$form_vars = [
 	'method' => 'get',
 	'action' => 'admin/administer_utilities/logbrowser',
 	'disable_security' => true,
-);
+];
 $form = elgg_view_form('logbrowser/refine', $form_vars, $vars);
 
-$toggle_link = elgg_view('output/url', array(
+$toggle_link = elgg_view('output/url', [
 	'href' => '#log-browser-search-form',
 	'text' => elgg_echo('logbrowser:search'),
 	'rel' => 'toggle',
-));
+]);
 
-$form_class = 'elgg-module elgg-module-inline';
+$toggle_link = elgg_format_element('div', [], $toggle_link);
+
+$module_options = ['id' => 'log-browser-search-form'];
 if (!isset($vars['user_guid']) && !isset($vars['username'])) {
-	$form_class .= ' hidden';
+	$module_options['class'] = 'hidden';
 }
 
-?>
+$module = elgg_view_module('inline', elgg_echo('logbrowser:search'), $form, $module_options);
 
-<div id="logbrowser-search-area" class="mbm">
```

Comment:
```php
<div id="logbrowser-search-area" class="mbm">
    <?= $toggle_link . $module ?>
</div>
```

Besides readability, PHPStorm can also see what's going on, recognize ids/classes in our stylesheets, etc.